spec/anchor-preview.spec.js

```diff
@@ -40,7 +40,7 @@ describe('Anchor Preview TestCase', function () {
             expect(anchorPreview.showPreview).toHaveBeenCalled();
 
             // link is set in preview
-            expect(anchorPreview.getPreviewElement().querySelector('i').innerHTML).toBe(document.getElementById('test-link').attributes.href.value);
+            expect(anchorPreview.getPreviewElement().querySelector(anchorPreview.previewValueSelector).innerHTML).toBe(document.getElementById('test-link').attributes.href.value);
```

Comment:
Well, I'm ok to use `a` but it feel more relevant to re-use the defined option (ie: `previewValueSelector`).

What if someone extends the AnchorPreview extension to change this selector without updating the spec.
Should the test pass or fail ?